### PR TITLE
Docs for XXE Injection Emulator

### PR DESCRIPTION
**Pull request by rjt-gupta**

### CODE REVIEW

**Comment by rjt-gupta** [comment]
@rnehra01 Can you look at the docker failure in the first commit's travis log. 

I think one possible fix is to pull the latest images before using `docker_compose up`.

**Comment by rnehra01** [comment]
The failing seems to be a problem with `alphine`. On running the `phpox` dockerfile locally, I'm getting
```
Step 3/6 : RUN apk -U --no-cache add                build-base                file                git                make                php7                php7-dev                php7-tokenizer                python3                python3-dev                re2c &&     pip3 install --no-cache-dir --upgrade pip &&     git clone --depth=1 https://github.com/mushorg/BFR /opt/BFR &&     cd /opt/BFR &&     phpize7 &&     ./configure       --with-php-config=/usr/bin/php-config7       --enable-bfr &&     make &&     make install &&     cd / &&     rm -rf /opt/BFR /tmp/* /var/tmp/* &&     echo "zend_extension = "$(find /usr -name bfr.so) >> /etc/php7/php.ini &&     git clone --depth=1 https://github.com/mushorg/phpox /opt/phpox &&     cd /opt/phpox &&     cp /root/dist/sandbox.py . &&     pip3 install -r requirements.txt &&     make &&     apk del --purge build-base                     git                     php7-dev                     python3-dev &&     rm -rf /root/* &&     rm -rf /var/cache/apk/*
 ---> Running in c75cf7a3eb89
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/armv7/APKINDEX.tar.gz
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.10/main/armv7/APKINDEX.tar.gz: temporary error (try again later)
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/armv7/APKINDEX.tar.gz
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.10/community/armv7/APKINDEX.tar.gz: temporary error (try again later)
ERROR: unsatisfiable constraints:
  build-base (missing):
    required by: world[build-base]
  file (missing):
    required by: world[file]
```
There seems to be a problem be `apk -U --no-cache add`. @glaslos any ideas to fix this?

**Comment by afeena** [comment]
Images should not be pulled, see https://github.com/mushorg/tanner/pull/338 for additional info about build error fix

**Comment by rjt-gupta** [comment]
Alternatively, using just `python3` instead of `3.6` also fixes the issue. But maybe @afeena 's fix is more robust.

**Comment by afeena** [comment]
@rjt-gupta I don't mind any solution until it works :) I created pr just to show you the direction, you can choose anything you like :)

**Comment by rjt-gupta** [comment]
> 
> 
> @rjt-gupta I don't mind any solution until it works :) I created pr just to show you the direction, you can choose anything you like :)

That's so nice of you. 
@rnehra01 What do you think?

**Comment by rnehra01** [comment]
> > @rjt-gupta I don't mind any solution until it works :) I created pr just to show you the direction, you can choose anything you like :)
> 
> That's so nice of you.
> @rnehra01 What do you think?

Yeah, no problem for me.

File: tanner/emulators/sqli.py [hunk 1]
```diff
@@ -42,9 +42,13 @@ def map_query(self, attack_value):
     async def get_sqli_result(self, attack_value, attacker_db):
         db_query = self.map_query(attack_value)
         if db_query is None:
-            error_result = 'You have an error in your SQL syntax; check the manual\
-                        that corresponds to your MySQL server version for the\
-                        right syntax to use near {} at line 1'.format(attack_value['id'])
+            if TannerConfig.get('SQLI', 'type') == 'MySQL':
```
Comment (rnehra01) [inline tanner/emulators/sqli.py]: You made this change [here](https://github.com/mushorg/tanner/pull/336/files)

Comment (rjt-gupta) [reply tanner/emulators/sqli.py]: I dont know how it got here. But I have Rebased.

**Comment by rjt-gupta** [comment]
Please merge this first.

**Pull request merged**